examples/docs/package.json

```diff
@@ -9,10 +9,10 @@
     "preview": "astro preview"
   },
   "dependencies": {
-    "@docsearch/react": "^1.0.0-alpha.27"
+    "@docsearch/react": "^3.0.0-alpha.41"
```

Comment:
ditto, this may need an update to documentation. Better to hold off for now and tackle separately.